### PR TITLE
f-cookie-banner@4.10.0 - Style updates.

### PR DESCRIPTION
**Pull request by kevinrodrigues**
### Added
- New styles changes to banner.
  - Button colour changes, positioning of buttons & font sizes.

Screenshot 2023-05-11 at 10 01 36
Screenshot 2023-05-11 at 10 01 48

### CODE REVIEW

File: packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue [hunk 1]
```diff
@@ -565,16 +558,36 @@ export default {
     }
 }
 
-.c-cookieBanner-cta {
+.c-cookieBanner-cta-container {
     padding-block: f.spacing(d) f.spacing(a);
     padding-inline: f.spacing(e);
 
+    .c-cookieBanner-cta {
+        width: 100%;
```
Comment (xander-marjoram) [inline packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue:566]: Please can you use logical properties instead, i.e., inline-size, margin-block-end, margin-inline-start.

File: packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue [hunk 2]
```diff
@@ -565,16 +558,36 @@ export default {
...
+        margin-bottom: f.spacing(d);
+    }
+
     @include f.media('>=mid') {
+        display: flex;
+        flex-direction: row-reverse;
         padding-block: f.spacing(d);
         padding-inline: f.spacing(d);
+        margin-inline: f.spacing(d);
         min-width: 320px;
```
Comment (xander-marjoram) [inline packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue:579]: This is one I missed originally, this should be min-inline-size

File: packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue (hunk 1 above)
Comment (kevinrodrigues) [reply packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue:566]: Fixed. https://github.com/justeattakeaway/fozzie-components/pull/2281 For some reason the branch update didn't push to the right place so i have a new pr.

File: packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue (hunk 2 above)
Comment (kevinrodrigues) [reply packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue:579]: fixed.

**Pull request closed without merging**